ETL/FactPatientMetrics/LoadFactPatientMetrics.sql

```diff
@@ -0,0 +1,17 @@
+
+-- etl/LoadFactPatientMetrics.sql
+INSERT INTO FactPatientMetrics (idAge,idPatient,idDate,Age,dateConsultation,totalConsults)
+SELECT
+DA.idAge AS'idAge' ,
```

Comment:
Replace the above line with this instead of having an `UPDATE` statement below which is basically the same
`CASE WHEN Age < 20 THEN Age ELSE 20 END AS 'idAge'`